Source/processcontainers/ProcessContainer.h

```diff
@@ -102,12 +102,10 @@ namespace ProcessContainers {
         // Stops the running containerized process. Returns true when stopped.
         // Note: if timeout == 0, call is asynchronous
         virtual bool Stop(const uint32_t timeout /*ms*/) = 0; 
-
-        virtual void AddRef() const = 0;
-        virtual uint32_t Release() = 0;
     };
 
-    struct IContainerAdministrator {
+    struct IContainerAdministrator 
```

Comment:
I'm not a big fan on it also but we follow the WebKit style guide where it is class/struct name { 
So could you please use that style all over the file(s) (I think Bram created a code beautifier script for this if that is more convenient)